atecc608a/main.c

```diff
@@ -77,17 +77,18 @@
     "configuration zone’s values. [y/n]: "
 
 /* Data used by tests */
-psa_key_slot_number_t atecc608a_private_key_slot = 0;
+psa_key_slot_number_t atecc608a_private_key_slot = 1;
 psa_key_slot_number_t atecc608a_public_key_slot = 9;
 
 enum {
     key_type = PSA_KEY_TYPE_ECC_PUBLIC_KEY(PSA_ECC_CURVE_SECP256R1),
-    keypair_type = PSA_KEY_TYPE_ECC_KEYPAIR(PSA_ECC_CURVE_SECP256R1),
-    key_bits = 256,
+    keypair_type = PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_CURVE_SECP256R1),
+    priv_bits = PSA_BYTES_TO_BITS(32),
+    pub_bits = PSA_BYTES_TO_BITS(64),
```

Comment:
No, `pub_bits` should be equal to `priv_bits`, and in fact there should be a single constant. A private key and the corresponding public key have the same bit size. The bit size is a characteristic of the mathematical object, not the size of the representation which depends on the representation format. For a SECP256R1 key, the key size is 256 bits.